projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -699,8 +699,16 @@ public function process_form_submission() {
 
 			// Process the content to populate Contact_Form::$last
 			if ( $post ) {
+				if ( str_contains( $post->post_content, '<!--nextpage-->' ) ) {
+					$postdata = generate_postdata( $post );
+					$page     = isset( $_POST['page'] ) ? sanitize_text_field( wp_unslash( $_POST['page'] ) ) : null; // phpcs:Ignore WordPress.Security.NonceVerification.Missing
+					$paged    = isset( $page ) ? $page : 1;
+					$content  = $postdata['pages'][ $paged - 1 ];
+				} else {
+					$content = $post->post_content;
+				}
```

Comment:
Not sure if we need any extra checks on any of this? ie, to confirm generate_postdata doesn't return false or $postdata['pages'] is defined or $postdata['pages'][ $paged - 1 ] gives a valid page value?